src/braket/pennylane_plugin/braket_device.py

```diff
@@ -288,6 +299,79 @@ def _tracking_data(task):
         else:
             return {"braket_failed_task_id": task.id}
 
+    def classical_shadow(self, obs, circuit):
```

Comment:
This probably should be moved to translation.py. Also, is this meant to be a general method, or for the AWS device only? There's another implementation in LocalQubitDevice.

Reply:
Can we move it, since it's a class method?

Reply:
I'm fine keeping it here, knowing that it overrides a PennyLane core method